bio_hansel/scripts/read_vcf.py

```diff
@@ -0,0 +1,27 @@
+import os
+import pandas as pd
+from typing import List, Dict
+
+
+def read_vcf(vcf_file: str) -> (pd.DataFrame):
+    """Reads in the generated vcf file, filters for 2-state SNVs and takes out unnecessary columns within the vcf file
+    Args: 
+    vcf_file: path to the vcf file
+
+    Returns:
+    df: returns the dataframe version of the generated vcf file from snippy
+    """
+    with open(vcf_file, 'r') as file:
+        for line in file:
+            line = line.strip()
+            if line.startswith('#CHROM'):
+                break
+    cols = line.split()
+
+    df = pd.read_table(
+        vcf_file, comment='#', header=None,
+        names=cols).rename(columns={'#CHROM': 'CHROM'})
+    df = df[df['ALT'].str.len() <= 1]
+    df = df.drop(['CHROM', 'ID', 'QUAL', 'FILTER', 'INFO', 'FORMAT'], 1)
```

Comment:
@gcttong This is a good idea and I wonder if we could go further and save the `POS` column to the index (e.g. `df.index = df.POS`) and create a df for just the alt/ref info and another df for the binary SNV state (basically a matrix of 0s and 1s). The index of each would be the same and it might make testing easier since you could test the `get_sequences` function with a very basic df with index of positions and 2 columns - ref and alt nucleotide characters. The functions working off the binary SNV state matrix/df could easily be tested with a small test df of 1s and 0s.